### PR TITLE
Functional Monitoring documentation update (5 of 5)

### PR DESCRIPTION
**Pull request by chris-campbell9**
Jira:
https://www.mulesoft.org/jira/browse/DOCS-11469

**DO NOT MERGE UNTIL RELEASE!**

### CODE REVIEW

File: modules/ROOT/pages/afm-upload-monitor.adoc [hunk 1]
```diff
@@ -21,33 +21,28 @@ You can upload a suite of tests that you have written and want to run as a monit
...
+For details on file structure and file requirements, see xref:bat-example-test-suite.adoc[Example of a Test Suite].
 
-== Result
+. Click *Upload Monitor*.
+. Click *Next*.
 
-The dashboard opens. The monitor is listed on the left. Beneath the name of the monitor is an indicator that explains when the current run of the monitor began. A line at the bottom of the list item shows how much time remains before the next run of the monitor. Also, the *Monitored Endpoints*, *Tests*, and *Schedules* sections will all be empty. After the monitor has run, the list item for the monitor will indicate whether the monitored endpoint passed or failed.
+== Result
 
+The monitor detail opens. An indicator displays how long the execution ran. The last executions list displays the test result.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-upload-monitor.adoc]: What is "the monitor detail"? Does this have a UI label? Is it "the monitor's detail screen"?

Comment (kbostrom) [reply modules/ROOT/pages/afm-upload-monitor.adoc]: Again with "an indicator": what is this, exactly?

File: modules/ROOT/pages/afm-upload-monitor.adoc [hunk 2]
```diff
@@ -21,33 +21,28 @@ You can upload a suite of tests that you have written and want to run as a monit
...
-* Whether the endpoint returned the expected code, as indicated by *PASSED* in the first column of the table
-* Statistics about the execution of the test of the endpoint
-
-As your monitor continues to run at the scheduled intervals, the statistics in this table are updated. They show the minimum, average, and maximum runtime for up to five executions of the test.
-
-Click *Tests* to see statistics and information pertaining to the monitor. The version number shown here is 0.0.1. However, if you upload the test suite again after making changes to it, you will see here a second card of information. The version number on this card will be 0.0.2.
+You can view a list of monitor schedules in the schedules table. You can also disable and enable schedules.
 
-Every subsequent version of the test suite that you upload will have its own card and that card will show the version of the monitor. If you delete the schedule for a version of the monitor, the card for that version will no longer appear in the *Tests* section.
+The last executions table and the average times chart will enable you to determine the following:
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-upload-monitor.adoc]: In most cases, if you end an intro. fragment with a colon, you don't need "following" wording:
```suggestion
The last executions table and the average times chart enable you to determine:
```

File: modules/ROOT/pages/afm-view-schedules-private-location.adoc [hunk 3]
```diff
@@ -5,18 +5,20 @@ endif::[]
 
 :imagesdir: ../assets/images
 
-You can find out how many schedules a private location is used by. When a private location is used by a schedule, a worker to run the test associated with the schedule is created in that location. Therefore, seeing the number of schedules that use a private location shows you how many workers are running in that location.
+You can determine the number of schedules that use a private location.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-view-schedules-private-location.adoc]: Ah, at first I thought that "determine" was the ability to configure rather than view....

Comment (kbostrom) [reply modules/ROOT/pages/afm-view-schedules-private-location.adoc]: How about...:
```suggestion
View the number of schedules that use a private location by navigating to that location.
```
??? Isn't this kind of gratuitous?

File: modules/ROOT/pages/afm-view-schedules-private-location.adoc [hunk 4]
```diff
@@ -5,18 +5,20 @@ endif::[]
...
-You can find out how many schedules a private location is used by. When a private location is used by a schedule, a worker to run the test associated with the schedule is created in that location. Therefore, seeing the number of schedules that use a private location shows you how many workers are running in that location.
+You can determine the number of schedules that use a private location. 
+
+When initially specifying the private location, a test run worker associated with the schedule is created in that location. Viewing the location shows how many workers are running.
 
 == Procedure
 
-. Display the list of locations by clicking the *Locations* tab at the top of the screen.
+. Display the list of locations by opening *Location Manager* from the homepage.
+. Tap the *Private Locations* tab.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-view-schedules-private-location.adoc]: click

File: modules/ROOT/pages/afm-view-schedules-private-location.adoc (hunk 3 above)
Comment (kbostrom) [reply modules/ROOT/pages/afm-view-schedules-private-location.adoc]: Yeah...this topic could be one sentence somewhere else. Something to look at when you really get into the weeds with this legacy stuff.

File: modules/ROOT/pages/afm-view-test-history.adoc [hunk 5]
```diff
@@ -1,31 +1,31 @@
...
-You can review the details of past executions of a test version.
+You can review the details of past test runs.
 
 == Procedure
 
-. Select the monitor for which you want to view a test version's historical results.
-. Click the *Tests* tab to see the list of test versions.
-. Click the History icon for the test version that you want to run.
+. Select the monitor for which you want to view test results.
+. Tap *View Monitor History* (image:afm-ui-history-button.png[width=1.3%,height=1.3%]) to view a list of recent test runs.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-view-test-history.adoc]: click

File: modules/ROOT/pages/afm-view-test-history.adoc [hunk 6]
```diff
@@ -1,31 +1,31 @@
...
 
 == Results
 
-The History page is opened for the test version. The page displays the status of each execution and statistics that are associated with it. The statistics are:
+The *History* page is opened for the test. The page displays the status of each execution as well as the following associated statistics:
 
-* When the test was last executed
+* The amount of endpoints passed from the total number of endpoints tested
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-view-test-history.adoc]:
```suggestion
* The number of endpoints passed from the total number of endpoints tested
```

File: modules/ROOT/pages/afm-view-test-history.adoc [hunk 7]
```diff
@@ -1,31 +1,31 @@
...
 * How long the execution lasted
+* The configuration associated with said execution
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-view-test-history.adoc]:
```suggestion
* The configuration associated with the execution
```

**Review by kbostrom — DISMISSED** [review]
CE

**Review by kbostrom — APPROVED** [review]
RA

**Review by IsaacEldridge — APPROVED** [review]
